OpenRA.Mods.Common/Lint/CheckTranslationReference.cs

```diff
@@ -240,6 +240,9 @@ void ILintPass.Run(Action<string> emitError, Action<string> emitWarning, ModData
 
 		void CheckUnusedKey(string key, string attribute, Action<string> emitWarning, string file)
 		{
+			if (key.StartsWith("color-", StringComparison.InvariantCulture))
+				return;
```

Comment:
this feels like the wrong. We are setting up a foot gun for ourselves in the far future, though atm I don't really have a better solution in mind

Reply:
Is it possible to have some kind of `[Attribute]` in the fluent files? To add an "ignore unused key" attribute.